### ISSUE TITLE
API: getClassScannerOptions returns null

### ISSUE THREAD

**Issue by Postremus** [SBPrime/AsyncWorldEdit-Premium#112]
Hello,

Example code:
```
IAsyncWorldEdit awe = (IAsyncWorldEdit)Bukkit.getPluginManager().getPlugin("AsyncWorldEdit");
System.out.println("awe: " + awe  == null); //false
System.out.println("classscanner: " + awe.getClassScannerOptions() == null); // true
```

The ClasscannerOptions are null. Is there anything special needed to do in order to get an instance?
For example, pasting using the api works.

```
AsyncWorldEditInjector version 3.4.3
AsyncWorldEdit version 3.4.6
```

Paper 1.10.2

Thank you
Postremus

**Comment by SBPrime** [comment]
1. Did the plugin finish loading?
2. What is the result for getAPI()

**Comment by Postremus** [comment]
1) Yes, it did finish loading. This code is executed with a command.
2) There is no method getApi() in IAsyncWorldEdit defined.
 I guess you mean getAPIVersion?
```
IAsyncWorldEdit awe = (IAsyncWorldEdit) Bukkit.getPluginManager().getPlugin("AsyncWorldEdit");
System.out.println("awe: " + awe == null); //false
System.out.println("classscanner: " + awe.getClassScannerOptions() == null); // true
System.out.println("api-version: " + awe.getAPIVersion()); // 3.002000005
```

**Comment by SBPrime** [comment]
There is a public method called getApi, but getAPIVersion will do. I'll check why you get null for the class scanner and get back to you.

**Comment by Postremus** [comment]
I guess (?) i found the problem. Look at AsyncWorlEditMain.getClassScannerOptions(). When the api is not null, return null..

**Comment by SBPrime** [comment]
Your right that might be the problem ;) I left the implementation of the IAsyncWorldEdit for compatibility and I use the getAPI from AsyncWorlEditMain.

**Comment by SBPrime** [comment]
Fixed in 3.4.7
